<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="newGameButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>New game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>761</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="amountOfDiskLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>21</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount of disks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ABButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>1. A -&gt; B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ACButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>570</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>2. A -&gt; C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BAButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>570</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>3. B -&gt; A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CBButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>570</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>6. C -&gt; B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BCButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>570</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>4. B -&gt; C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CAButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>570</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>5. C -&gt; A</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="diskAmountSpinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>49</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="winLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="illegalLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="gameLogTextBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="gameLogLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Game log:</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentTimelabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>630</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Current game</string>
    </property>
   </widget>
   <widget class="QLabel" name="bestTimeLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>660</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Best time</string>
    </property>
   </widget>
   <widget class="QLabel" name="timerLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>610</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Timer</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="bestMinLcdNumber">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>660</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="bestSecLcdNumber">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>660</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="currentSecLcdNumber">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>630</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="currentMinLcdNumber">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>630</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
